CMakeLists.txt

```diff
@@ -501,6 +505,11 @@ while(library_variants_copy)
 
     set(runtimes_flags "${common_flags} -ffunction-sections -fdata-sections -fno-ident --sysroot ${install_directory}")
 
+    set(compiler_rt_extra_flags)
```

Comment:
It would be slightly neater to set this to `${runtimes_flags}`
This would simplify the ExternalProject_Add statement a little.

Reply:
Now not required, since I've removed that entire mechanism.